net/grpc/gateway/examples/echo/BUILD.bazel

```diff
@@ -9,22 +10,90 @@ proto_library(
     ],
 )
 
+# Server
```

Comment:
Quick question: any changes here will be tested by this https://github.com/grpc/grpc-web/blob/master/scripts/docker-run-tests.sh#L22 that's being executed by our CI right? If that's the case, then we will get reassurances that these will get continuously tested.

Reply:
Yes, everything here is built on CI.